common/schema/schema_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	"github.com/alecthomas/assert/v2"
+	"golang.org/x/exp/maps"
```

Comment:
FYI this is in the stdlib as "maps" now

Reply:
It returns `iter.Seq` though, so I am not sure it would work in this case.